src/azure-cli/azure/cli/command_modules/apim/_params.py

```diff
@@ -114,6 +114,31 @@ def load_arguments(self, _):
         c.argument('subscription_required', arg_group='API', arg_type=get_three_state_flag(), help='If true, the API requires a subscription key on requests.')
         c.argument('tags', tags_type)
 
+    with self.argument_context('apim nv create') as c:
+        c.argument('service_name', options_list=['--service-name'], help='The name of the API Management service instance.')
+        c.argument('named_value_id', help='The id of the API Management Named Value.')
+        c.argument('display_name', help='The display name of the API Management Named Value.')
+        c.argument('value', help='The value of the API Management Named Value.')
+        c.argument('secret', arg_type=get_three_state_flag(), help='If true, the Named Value is a secret.')
+        c.argument('tags', tags_type)
+
+    with self.argument_context('apim nv update') as c:
```

Comment:
You can put `c.argument('service_name')` under `with self.argument_context('apim nv')`, and we can share it across all sub command.

Reply:
maybe just delete the left part and rename `apim nv create` to `apim nv`.

Reply:
okay I will do so.